grails-app/services/au/org/ala/bie/ImportService.groovy

```diff
@@ -1589,30 +1750,36 @@ class ImportService {
     /**
      * Collect the list where images are specifically listed
      */
-    def collectImageLists() {
+    def collectImageLists(List lists) {
         def speciesListUrl = grailsApplication.config.speciesList.url
         def speciesListParams = grailsApplication.config.speciesList.params
         def imageMap = [:]
         log("Loading image lists")
-        grailsApplication.config.imageLists.each { list ->
-            String drUid = list.drUid
+        lists.each { list ->
+            String drUid = list.uid
             String imageIdName = list.imageId
             String imageUrlName = list.imageUrl
             if (drUid && (imageIdName || imageUrlName)) {
                 def url = "${speciesListUrl}${drUid}${speciesListParams}"
-                JSONElement json = JSON.parse(getStringForUrl(url))
-                json.each { item ->
-                    def taxonID = item.lsid
-                    def name = item.name
-                    def imageId = imageIdName ? item.kvpValues.find { it.key == imageIdName }?.get("value") : null
-                    def imageUrl = imageUrlName ? item.kvpValues.find { it.key == imageUrlName }?.get("value") : null
-                    if (imageId || imageUrl) {
-                        def image = [taxonID: taxonID, name: name, imageId: imageId, imageUrl: imageUrl]
-                        if (taxonID && !imageMap.containsKey(taxonID))
-                            imageMap[taxonID] = image
-                        if (name && !imageMap.containsKey(name))
-                            imageMap[name] = image
+                try {
+                    JSONElement json = JSON.parse(getStringForUrl(url))
+                    json.each { item ->
+                        def taxonID = item.lsid
+                        def name = item.name
+                        def imageId = imageIdName ? item.kvpValues.find { it.key == imageIdName }?.get("value") : null
+                        def imageUrl = imageUrlName ? item.kvpValues.find {
+                            it.key == imageUrlName
+                        }?.get("value") : null
+                        if (imageId || imageUrl) {
+                            def image = [taxonID: taxonID, name: name, imageId: imageId, imageUrl: imageUrl]
+                            if (taxonID && !imageMap.containsKey(taxonID))
+                                imageMap[taxonID] = image
+                            if (name && !imageMap.containsKey(name))
+                                imageMap[name] = image
+                        }
                     }
+                } catch (Exception ex) {
+                    log("Unable to load image list at ${url}: ${ex.getMessage()} ... ignoring")
```

Comment:
Fair enough - personally, I generally prefer to always have the stacktrace (because it's easy to ignore it when you don't care, but it's really irritating if it isn't there when you need it).